modules/rest-api/pages/rest-set-up-services-existing-nodes.adoc

```diff
@@ -0,0 +1,150 @@
+
+* `topology`: The value is a list of services and the nodes in which services must reside, after the rebalance operation completes.
++
+The value is `topology[<service>]=<list of nodes>`.
++
+Where,
+
+** `<service>` can be one of the following services:
+
+*** `fts` for the Search Service.
+*** `index` for the Index Service.
+*** `n1ql` for the Query Service.
+*** `backup` for the Backup Service.
+*** `cbas` for the Analytics Service.
+
+** `<list of nodes>` indicate the nodes on which the `<service>` must reside in after the rebalance operation is complete.
+The node names are separated by commas.
++
+If a `<service>` is running on any node in the cluster that's not in this list, Couchbase Server removes that `<service>` during the rebalance operation.
+For more information, see xref:rest-api:rest-cluster-rebalance.adoc[Rebalancing the Cluster].
+
+** For example, `-d topology[<service>]=otp_node1,otp_node2`.
+
+NOTE: The rebalance occurs even if no `<service>` is changed.
+
+Each service you assign uses the memory quota currently configured for that service on the cluster.
+You can modify the services memory quotas, if needed, prior to running the services reconfiguration command for the cluster nodes.
+
+For information about allocating memory per service, see xref:rest-api:rest-configure-memory.adoc[Configuring Memory].
+
+To fetch the details of current memory allocation for each service, see xref:rest-api:rest-get-memory-information.adoc[Getting Memory Information].
+
+== Required Permissions
+
+`Full Admin` role.
+
+[#responses]
+== Responses
+
+|===
+|Value | Description  
+
+| `200 OK` and JSON array containing the expired backups and their details.
+| Successful call.
+
+| `400`
+| Invalid parameter.
+
+| `400 Object Not found`
+| The repository in the endpoint URI does not exist.
+
+| `401 Unauthorized`
+|  Authorization failure due to incorrect username or password.
+
+| `403 Forbidden`, plus a JSON message explaining the minimum permissions.
+| The provided username has insufficient privileges to call this method.
+
+| `404 Object Not Found`
+| Error in the URI path.
+
+| `500 Internal Server Error`
+| Error in Couchbase Server.
+|===
+
+=== Example
+
+For example, a cluster with 5 nodes could have the following service configuration:
+
+* 172.23.108.69 – Data, Index, Query
+* 172.23.108.70 – Data, Search, Query
+* 172.23.108.71 – Query
+* 172.23.108.72 – Backup
+* 172.23.108.73 – Analytics
+
+You could reconfigure the services on the existing nodes to match the following:
+
+* 172.23.108.69 – Data, Query
+* 172.23.108.70 – Data, Index
+* 172.23.108.71 – Index, Query
+* 172.23.108.72 – Search, Query
+* 172.23.108.73 – Analytics
+
+Because the Backup service is not in the topology list, it will be removed from the cluster.
+After the topology change and rebalance, two nodes (172.23.108.71 and 172.23.108.72) that previously had no services using the index disk storage path will now have services using it.
+
```

Comment:
Sep 22, 2025 update -- add this additional info:
Also, in the command shown below, the Backup Service is being removed from the cluster, as the `-d topology[backup]=` does not have any nodes listed.